config-qinstall.yml

```diff
@@ -0,0 +1,356 @@
+    mountpoint: /anfhome # /sharedhome for example
+    server: '{{anf_home_ip}}' # Specify an existing NFS server name or IP, when using the ANF built in use '{{anf_home_ip}}'
+    export: '{{anf_home_path}}' # Specify an existing NFS export directory, when using the ANF built in use '{{anf_home_path}}'
+    options: '{{anf_home_opts}}' # Specify the mount options. Default to rw,hard,rsize=262144,wsize=262144,vers=3,tcp,_netdev
+
+# name of the admin account
+admin_user: hpcadmin
+# List of identities (object ids) to grant read access to az-hop key vault (optional)
+# key_vault_readers:
+# Network
+network:
+  # Create Network and Application Security Rules, true by default, false when using an existing VNET if not specified
+  create_nsg: true
+  vnet:
+    name: hpcvnet # Optional - default to hpcvnet
+    #id: # If a vnet id is set then no network will be created and the provided vnet will be used
+    address_space: "10.0.0.0/23" 
+    # When using an existing VNET, only the subnet names will be used and not the adress_prefixes
+    subnets: # all subnets are optionals
+    # name values can be used to rename the default to specific names, address_prefixes to change the IP ranges to be used
+    # All values below are the default values
+      frontend:
+        name: frontend
+        address_prefixes: "10.0.0.0/29"
+        create: true # create the subnet if true. default to true when not specified, default to false if using an existing VNET when not specified
+      admin:
+        name: admin
+        address_prefixes: "10.0.0.16/28"
+        create: true
+      netapp:
+        name: netapp
+        address_prefixes: "10.0.0.32/28"
+        create: true
+      ad:
+        name: ad
+        address_prefixes: "10.0.0.8/29"
+        create: true
+      compute:
+        name: compute
+        address_prefixes: "10.0.1.0/24"
+        create: true
+  # Specify the Application Security Groups mapping if already existing
+
+# When working in a locked down network, uncomment and fill out this section
+locked_down_network:
+  enforce: false
+#   grant_access_from: [a.b.c.d] # Array of CIDR to grant access from, see https://docs.microsoft.com/en-us/azure/storage/common/storage-network-security?tabs=azure-portal#grant-access-from-an-internet-ip-range
+  public_ip: true # Enable public IP creation for Jumpbox, OnDemand and create images. Default to true
+
+# Base image configuration. Can be either an image reference or an image_id from the image registry or a custom managed image
+linux_base_image: "OpenLogic:CentOS:7_9-gen2:latest" # publisher:offer:sku:version or image_id
+# linux image plan if required, format is publisher:product:name
+#linux_base_plan:
+windows_base_image: "MicrosoftWindowsServer:WindowsServer:2019-Datacenter-smalldisk:latest" # publisher:offer:sku:version or image_id
+lustre_base_image: "azhpc:azurehpc-lustre:azurehpc-lustre-2_12:latest"
```

Comment:
both lustre properties should be removed

Reply:
Removed